client/js/verification-portal/community-health-center/util.js

```diff
@@ -0,0 +1,26 @@
+export const normalizePhone = input => {
```

Comment:
@bhavyaab We shouldn't duplicate this code. Let's move [this function](https://github.com/HelpAssistHer/help-assist-her/blob/master/client/js/verification-portal/pregnancy-resource-center/form.js#L13) into a util file and use the same function in both places. Ideally, that work would be done in a separate PR so that we can test to make sure it still works for the existing functionality (the PRC form) and then merge that work into this PR.

Reply:
Will do it in separate pr after finishing this. will that be ok?

Reply:
@bhavyaab I would prefer if we don't merge this code, but commit the clean and working code that we'll keep. If it helps, we can do this in a separate PR, get that merged, and then merge it into this PR.